.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,79 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ master, dev-build ]
+
+env:
+  ORG: timescale # timescaledev
+  
+jobs:
+
+  # Build multi-arch TimescaleDB images of TSL and OSS code
+  timescaledb:
+
+    name: PG${{ matrix.pg }}${{ matrix.oss }}
+    runs-on: ubuntu-latest
+    strategy: 
+      matrix:
+        pg: [ 9.6, 10, 11, 12 ]
+        oss: [ "", "-oss" ]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: crazy-max/ghaction-docker-buildx@v2
+      with:
+        buildx-version: latest
+        skip-cache: false
+        qemu-version: latest
+
+    - name: Available platforms
+      run: echo ${{ steps.buildx.outputs.platforms }}
+
+    - name: Login to DockerHub Registry
+      run: echo ${{ secrets.DOCKERHUB_TOKEN }} | docker login -u ${{ secrets.DOCKERHUB_USER }} --password-stdin
+
+    - name: Build and push multi-platform Docker image for TimescaleDB
+      run: make multi${{ matrix.oss }} ORG=$ORG PG_VER=pg${{ matrix.pg }}
+
+  # Build bitnami images of TimscaleDB, single arch and TSL only
```

Comment:
```suggestion
  # Build TimescaleDB based off of Bitnami base images. The base images only exists for amd64, so we only build for a single architecture here. Currently we don't build an OSS version of these images since that is currently provided by the standard TimescaleDB image. 
```